test/part1.js

```diff
@@ -0,0 +1,51 @@
+import Data from './__mock__/mockData.js';
+
+export const addNew = (description) => {
+  const data = new Data();
+  const tasks = data.getData();
+  const newTask = {
+    index: tasks.length + 1,
+    description,
+    completed: false,
+  };
+
+  if (newTask.description.length > 0) {
+    tasks.push(newTask);
+    data.setData(tasks);
+  }
+
+  if (newTask.description.length > 0) {
+    const check = data.getData();
+    return check[check.length - 1];
+  }
+  throw new Error('task cannot be an empty string');
+};
+
+export const deleteTasks = (checks) => {
+  // const checks = document.querySelectorAll('input[type="checkbox"]');
```

Comment:
- [ ] Let's endeavour to remove commented code so as to avoid confusion in reading the code.

Reply:
Hello @TedLivist ,
Thanks for your your suggestion.